content/en/tracing/guide/setting_primary_tags_to_scope.md

```diff
@@ -83,10 +83,79 @@ If you change a previously set primary tag, be aware of the following:
 * Historical APM data aggregated by the previously set tag is no longer accessible.
 * Any APM monitors scoped to the previous tag display a status of _No Data_.
 
+## Container-based second primary tags
 
-### Data by primary tag
+You can index your trace metrics based on the tags derived from Docker containers and Kubernetes pod metadata on Linux-based platforms. Container-based second primary tags are available in Datadog Agent versions 7.35.0 and later.
 
-Primary tags appear at the top of APM pages. Use these selectors to slice the data displayed on the current page. To view all data independent of a primary tag, choose `<TAG_NAME>:*` from the dropdown.
+To enable container-based second primary tags, install Agent version 7.35.0 or later, update the CID stats setting as described below, and restart the Agent. The procedure for enabling depends on how you installed the Agent:
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+Using the Datadog Helm chart version 2.26.2 or later, and add the following to your values file:
+
+```yaml
+#...
+datadog:
+  #...
+  env:
+    - name: DD_APM_FEATURES
+      value: 'enable_cid_stats'
+```
+
+{{% /tab %}}
+
+{{% tab "Kubernetes (without Helm)" %}}
+
+Use the following environment variable in the Agent DaemonSet. If you are running a container per Agent process, add the following environment variable to all containers. Otherwise, add it to the Agent container.
+
+```yaml
+# (...)
+  env:
+    # (...)
+    - name: DD_APM_FEATURES
+      value: 'enable_cid_stats'
+```
+
+{{% /tab %}}
+{{% tab "Docker Compose" %}}
+
+Add the following to your [docker-compose.yml][1] file:
+
+```yaml
+services:
+  #...
+  datadog:
+    #...
+    environment:
+     - DD_APM_FEATURES: 'enable_cid_stats'
+```
+
+
+[1]: /agent/guide/compose-and-the-datadog-agent/
+{{% /tab %}}
+{{% tab "Environment variables" %}}
+
+If you configure the Agent with environment variables, as is common with Docker and ECS installations, pass the following environment variable to the trace Agent after upgrading the Docker image.
+
+```
+DD_APM_FEATURES=enable_cid_stats
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Restart the Agent. Go to the [APM Settings][6] page and select the second primary tag you want to use. It can take up to two hours for changes to this setting to take effect. 
+
+Now you can filter your services in the [Service List][7] by the tag being sent by your containerized services. Trace metrics used by Dashboards and Monitors can also be aggregated by the container primary tag.
+
+### Custom Labels as Tags
```

Comment:
Echoing the Vale warning about sentence-style capitalization for headings... Are custom labels and tags both proper nouns in this context?